### PR TITLE
Fixed German Spelling

### PR DESCRIPTION
**Pull request by Nilia119**
Fixed German Spelling.

### CODE REVIEW

File: addons/chemical/stringtable.xml [hunk 1]
```diff
@@ -330,8 +330,8 @@
             <Italian>M43 BZ LGB</Italian>
         </Key>
         <Key ID="STR_kat_chemical_M43_desc">
-            <English>Laser Guided Cluster Bomb with an chemical gas payload.</English>
-            <German>Laser gelenkte Senfgas Cluster Bombe.</German>
+            <English>Laser Guided Cluster Bomb with a chemical gas payload.</English>
+            <German>Laser gelenkte Streubombe mit Giftgas.</German>
```
Comment (AtrixZockt) [inline addons/chemical/stringtable.xml]: Why do you change senfgas to giftgas?

File: addons/chemical/stringtable.xml [hunk 2]
```diff
@@ -355,7 +355,7 @@
         </Key>
         <Key ID="STR_kat_chemical_HydraGas_desc">
             <English>Unguided rockets with Toxic-gas warhead.</English>
-            <German>Ungelengte Racketen mit Senfgas sprengkopf.</German>
+            <German>Ungelenkte Raketen mit Giftgas-Sprengkopf.</German>
```
Comment (AtrixZockt) [inline addons/chemical/stringtable.xml]: same here

File: addons/chemical/stringtable.xml [hunk 3]
```diff
@@ -259,23 +259,23 @@
...
+            <German>Giftgas</German>
             <French>Gaz toxique</French>
             <Japanese>有毒ガス</Japanese>
             <Korean>유독가스</Korean>
             <Italian>Gas tossico</Italian>
         </Key>
         <Key ID="STR_kat_chemical_ToxicGas_DisplayName">
             <English>82mm Toxic Gas Round</English>
-            <German>82mm Senf Gas Patrone</German>
+            <German>82mm Giftgasgranate</German>
```
Comment (AtrixZockt) [inline addons/chemical/stringtable.xml]: senfgas to giftgas?

**Comment by MiszczuZPolski** [comment]
look at #257, if something need fix then update this PR and tell us to check it

**Comment by Nilia119** [comment]
Updated it to Current state.

File: addons/chemical/stringtable.xml [hunk 4]
```diff
@@ -426,7 +426,7 @@
         </Key>
         <Key ID="STR_kat_chemical_LvL1_Gas">
             <English>Toxic gas (Level 1)</English>
-            <German>Toxic gas (Level 1)</German>
+            <German>Giftgas (Level 1)</German>
```
Comment (AtrixZockt) [inline addons/chemical/stringtable.xml]: Pls change Giftgas to Senfgas since i made the mistake and did not change every toxic gas to senfgas

**Review by AtrixZockt — CHANGES_REQUESTED** [review]
expect the 1 thing i'll approve

File: addons/chemical/stringtable.xml (hunk 4 above)
Comment (Nilia119) [reply addons/chemical/stringtable.xml]: Sorry, i changed it.

**Review by AtrixZockt — APPROVED** [review]

**Pull request merged**